app/components/shared/sdg/goal_stats_component.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="sdg-goal-stats">
+
+  <h3><%= goal.code_and_title %></h3>
+
+  <div class="stats-numbers">
+
+    <div class="small-12 medium-4 column">
+      <p>
+        <%= t("sdg.stats.polls") %> <br><span class="number"><%= goal.polls.count %></span>
+      </p>
+    </div>
+
+    <div class="small-12 medium-4 column">
+      <p>
+        <%= t("sdg.stats.proposals") %> <br><span class="number"><%= goal.proposals.count %></span>
+      </p>
+    </div>
+
+    <div class="small-12 medium-4 column">
+      <p>
+        <%= t("sdg.stats.debates") %> <br><span class="number"><%= goal.debates.count %></span>
```

Comment:
Here, I wanted to use the next format:
```
<%= t("sdg.stats.debates") %> <br>
<span class="number"><%= goal.debates.count %></span>
```
the same we are using at `admin/stats/show.html.erb`, but the [test](https://github.com/consul/consul/pull/4323/files#diff-945a664114cd9aafbbb3111efcc100260c21aca2ae6ee85c861ffc4e106fad14R16) failed because the `page.text` has the text `Proposals         \n3` so the following expectations fails:

```
expect(page).to have_text "Proposals 3"
```
@javierm, @taitus do you know what I am missing here? Maybe this is something related to the type of the spec, which is `:component`.

Reply:
The `system/admi/stats_spec.rb` tests are passing because they've got JavaScript disabled. It looks like component specs follow similar rules as JavaScript tests :thinking:.

So, yeah, probably the test should expect the test to have the "\n" :thinking:.